alpha/hal.h

```diff
@@ -104,6 +111,7 @@ class _TS_HAL
     //
     void led_set(TS_Led, bool);
     bool btn_a_get();
+    TS_ButtonState btn_power_get();
```

Comment:
could you add in btn_b as well for completeness? should be pretty trivial

Reply:
yup sure I'll make the changes together with moving stuff under HAL